blog/.gitignore

```diff
@@ -1,2 +1,3 @@
 _site
 .sass-cache
+.jekyll-metadata
```

Comment:
Yes, that's the output directory.  Agree, I'll add it to .gitignore, that will also prevent committing of built blog posts in case users accidentally run Jekyll build on their machines.  In the past, I think the committing of multiple output files was causing some strange issues / merge conflicts.